inventory-repository/src/main/resources/ldap/inventory.schema

```diff
@@ -974,7 +983,7 @@ objectclass ( InventoryObjectClasses:14
         inventoryDate $ receiveDate $ ipHostNumber $ osFullName $
         osArchitectureType $ lastLoggedUser $ lastLoggedUserTime $
         environmentVariable $ process $ rudderServerRole $ keyStatus $
-        timezoneName $ timezoneOffset ) )
+        timezoneName $ timezoneOffset $ customProperty) )
```

Comment:
shouldn't it also trigger a change of inventory-repository/src/test/resources/ldap-data/schema/099-0-inventory.ldif
and
rudder/rudder-core/src/test/resources/ldap-data/schema/099-0-inventory.ldif
?